code/modules/mining/aux_base.dm

```diff
@@ -144,7 +144,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/auxiliary_base, 32)
 				return
 			var/shuttle_error = SSshuttle.moveShuttle(shuttleId, params["shuttle_id"], 1)
 			if(launch_warning)
-				say(span_danger("Launch sequence activated! Prepare for drop!!"))
+				audible_message(span_danger("Launch sequence activated! Prepare for drop!!"))
```

Comment:
Keep it as a say, and just remove the span from around the message.

Reply:
I want to preserve the span class of the message since its warning and all

Reply:
Then pass the danger span into the say.

`say(>>insert message here<<, spans = list(danger))